i18n/en.json

```diff
@@ -4096,7 +3572,7 @@
   },
   {
     "id": "model.client.get_team_icon.app_error",
-    "translation": ""
+    "translation": "Unable to read the team icon from the body response."
```

Comment:
What is the body response in this case?

Reply:
This error exists in the Go Client library of the API, this means that a developer using the go client library to get the team icon through the API, can receive a body which it can read (for any reason).